content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/mto.rst

```diff
@@ -2,121 +2,127 @@
+are currently archived. Tick the checkbox next to :guilabel:`Replenish on Order (MTO)`, then click
+the :icon:`fa-cog` :guilabel:`Actions` button to reveal a drop-down menu. From the drop-down menu,
+select :guilabel:`Unarchive`.
 
 .. image:: mto/unarchive-button.png
    :align: center
    :alt: The unarchive action on the Routes page.
 
-Finally, remove the :guilabel:`Archived` filter from the :guilabel:`Search...` bar. The
-:guilabel:`Routes` page will now show all available routes, including :guilabel:`Replenish on Order
-(MTO)`, which is now selectable on the inventory tab of each product page.
+Finally, remove the :guilabel:`Archived` filter from the search bar. The :guilabel:`Routes` page now
+shows all unarchived routes, including :guilabel:`Replenish on Order (MTO)`, which is selectable on
+the *Inventory* tab of each product page.
 
-.. image:: mto/unarchived-mto.png
-   :align: center
-   :alt: The MTO route appears on the Routes page after unarchiving it.
-
-Configure a product to use the MTO route
-========================================
+Configure product for MTO
+=========================
 
 With the |MTO| route unarchived, products can now be properly configured to use replenish on order.
-To do so, begin by going to :menuselection:`Inventory --> Products --> Products`, then select an
-existing product, or click :guilabel:`Create` to configure a new one.
+To do so, begin by going to :menuselection:`Inventory app --> Products --> Products`, then select an
+existing product, or click :guilabel:`New` to configure a new one.
 
 On the product page, select the :guilabel:`Inventory` tab and enable the :guilabel:`Replenish on
-Order (MTO)` route in the :guilabel:`Routes` section, along with one other route.
+Order (MTO)` route in the :guilabel:`Routes` section, along with the :guilabel:`Buy` or
+:guilabel:`Manufacture` route.
 
 .. important::
    The :guilabel:`Replenish on Order (MTO)` route **does not** work unless another route is selected
    as well. This is because Odoo needs to know how to replenish the product when an order is placed
-   for it (buy it, manufacture it, etc.).
+   for it (buy or manufacture it).
 
 .. image:: mto/select-routes.png
    :align: center
    :alt: Select the MTO route and a second route on the Inventory tab.
 
-If the product is purchased from a vendor to fulfill sales orders, enable the :guilabel:`Can be
-Purchased` checkbox under the product name. Doing so makes the :guilabel:`Purchase` tab appear
-alongside the other settings tabs below.
+If the product is purchased from a vendor to fulfill |SOs|, enable the :guilabel:`Can be Purchased`
+checkbox under the product name. Doing so makes the :guilabel:`Purchase` tab appear alongside the
+other tabs below.
 
-Select the :guilabel:`Purchase` tab and specify a :guilabel:`Vendor` and the :guilabel:`Price` they
+Click the :guilabel:`Purchase` tab and specify a :guilabel:`Vendor` and the :guilabel:`Price` they
 sell the product for.
```

Comment:
IIRC *not* doing this creates an error on RFQs, right? If so, might be worth emphasizing the importance of doing this.